test/recipes/60-test_x509_check_cert_pkey.t

```diff
@@ -60,4 +60,6 @@ ok(run(test(["x509_check_cert_pkey_test",
              src_file("wrongkey.pem"), "req", "failed"])));
 
 test_PEM_X509_INFO_read("root-cert.pem", "1");
+test_PEM_X509_INFO_read("root-key.pem", "1");
+test_PEM_X509_INFO_read("key_pass_12345.pem", "1");
```

Comment:
As other key files use `-` in names rather than `_` perhaps it would be more consistent to use dashes here as well.

Reply:
Yep, done.